src/OpenCV/ImgProc/ImgFiltering.hsc

```diff
@@ -296,6 +297,36 @@ blur size matIn =
   where ksize :: Size2i
         ksize = toSize2i size
 
+gaussianBlur
```

Comment:
Note that here, `sigmaX` and `sigmaY` can be omitted if the kernel is provided, and vice-versa. However I think they can all be provided at once as well. Not sure what the convention is in haskell-opencv is in these cases.